src/components/pages/career-details/career-form/career-file-upload-type.tsx

```diff
@@ -82,8 +82,13 @@ export const CareerFileUploadType = ({
   watch,
 }: CareerFileUploadTypeProps) => {
   const { t } = useTranslation();
-  const fileName = file.name.split('.');
-  const name = `category_select.${fileName[0]}`;
+  const fileName = file.name.split('.')[0];
+  /**
+   * Create a sanitized file id by replacing anything that's not a letter which can be safely used as an identifier.
+   * Without this, a valid filename like `a,b.pdf` (osx) will create an identifier to safely use as an internal name.
```

Comment:
I don't understand this sentence...

Reply:
Early morning fix. I will update it. Sorry.